packages/react/src/components/Carousel/Carousel.tsx

```diff
@@ -270,14 +273,23 @@ class Carousel extends AutoControlledComponent<WithAsProp<CarouselProps>, Carous
           )}
         >
           {items &&
-            items.map((item, index) => {
-              const itemRef = React.createRef<HTMLElement>()
-              this.itemRefs.push(itemRef)
-              return (
+          items.map((item, index) => {
+            const itemRef = React.createRef<HTMLElement>()
+            this.itemRefs.push(itemRef)
+            const active = activeIndex === index
+            const slideToNext = prevActiveIndex < activeIndex
+            return (
+              <Animation
+                mountOnEnter
+                unmountOnExit
+                visible={active}
+                // TODO figure out which animations should be used and add required names in the theme
+                name={active ? slideToNext ? 'carousel-slide-to-next-enter' : 'carousel-slide-to-previous-enter' : slideToNext ? 'carousel-slide-to-next-exit' : 'carousel-slide-to-previous-exit'}
```

Comment:
@codepretty @bhamlefty can you guys suggest which animations we should use on the carousel for sliding to next/prev  items?